contracts/DAIInterestRateModelV3.sol

```diff
@@ -42,12 +42,16 @@ contract DAIInterestRateModelV3 is JumpRateModelV2 {
     }
 
     /**
-     * @notice Update the gapPerBlock parameter of the interest rate model (only callable by owner, i.e. Timelock)
-     * @param gapPerYear The additional margin per year separating the base borrow rate from the roof.
+     * @notice External function to update the parameters of the interest rate model
+     * @param baseRatePerYear The approximate target base APR, as a mantissa (scaled by 1e18). For DAI, this is calculated from DSR and SF. Input not used.
+     * @param gapPerYear The Additional margin per year separating the base borrow rate from the roof. (scaled by 1e18)
+     * @param jumpMultiplierPerYear The jumpMultiplierPerYear after hitting a specified utilization point
+     * @param kink_ The utilization point at which the jump multiplier is applied
      */
-    function updateGap(uint gapPerYear) public {
+    function updateJumpRateModel(uint baseRatePerYear, uint gapPerYear, uint jumpMultiplierPerYear, uint kink_) external {
         require(msg.sender == owner, "only the owner may call this function.");
-        gapPerBlock = gapPerYear / blocksPerYear;
+        gapPerBlock = gapPerYear.div(blocksPerYear);
+        updateJumpRateModelInternal(0, 0, jumpMultiplierPerYear, kink_);
```

Comment:
Think this is preferable to `super.updateJumpRateModel`?

Reply:
I think it's a little less redundant don't you? Also, I made updateJumpRateModel external to save some gas so we won't be able to do it like that without changing back.

Reply:
yeah like it with the internal myself as well!